addons/project/tests/test_project_sharing_portal_access.py

```diff
@@ -62,15 +62,6 @@ def test_mention_suggestions(self):
             {self.user_projectuser.partner_id.id, self.user_portal.partner_id.id},
             "Portal user as a project collaborator should have access to mention suggestions",
         )
-        # remove portal user from the project collaborators
-        self.project_portal.collaborator_ids.filtered(
-            lambda rec: rec.partner_id == self.user_portal.partner_id
-        ).unlink()
-        self.assertEqual(
-            {},
-            self.task_portal.with_user(self.user_portal).get_mention_suggestions(search=""),
-            "Non collaborator portal user should not have access to mention suggestions",
-        )
```

Comment:
I don't understand why you remove that code. 🤔 

Can you explain?

Reply:
I previously removed this code as it was related to the following : [changes](https://github.com/odoo/odoo/pull/157332/files#diff-813df946c751de1ce26749a2c1fcf76f3a453f71629724824eb4a6b9a29abfd1R34). However, the requirements have since changed, and it is no longer necessary to exclude this code. As a result, I’ve re-added it.